code/game/objects/machinery/doors/windowdoor.dm

```diff
@@ -86,6 +86,9 @@
 	. = ..()
 	if(istype(mover) && CHECK_BITFIELD(mover.flags_pass, PASSGLASS))
 		return TRUE
+	if(get_dir(loc, target) == dir) //Make sure looking at appropriate border
+		return
```

Comment:
```suggestion
		return FALSE
```

Reply:
This seems to be the common return for "cannot" pass through.

Reply:
This suggestion is wrong